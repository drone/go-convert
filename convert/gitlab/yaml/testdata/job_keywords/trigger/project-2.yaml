# https://docs.gitlab.com/ee/ci/yaml/#triggerproject

trigger-multi-project-pipeline:
  trigger:
    project: my-group/my-project
    branch: development