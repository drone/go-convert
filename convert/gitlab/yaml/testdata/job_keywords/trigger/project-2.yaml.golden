trigger-multi-project-pipeline:
    trigger:
        project: my-group/my-project
        branch: development
